<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1536</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPlainTextEdit" name="editA">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>491</width>
     <height>641</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="editB">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>491</width>
     <height>641</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="editC">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>10</y>
     <width>491</width>
     <height>641</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_encode">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>690</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>aes256Encrypt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_decode">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>680</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>aes256Decrypt</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
